smoketest/scripts/cli/test_policy.py

```diff
@@ -1116,5 +1116,58 @@ def test_table_id(self):
 
         self.assertEqual(tmp, original)
 
+    # Test set table for fwmark
+    def test_fwmark_table_id(self):
+        path = base_path + ['local-route']
+
+        fwmk = '24'
+        rule = '101'
+        table = '154'
+
+        self.cli_set(path + ['rule', rule, 'set', 'table', table])
+        self.cli_set(path + ['rule', rule, 'fwmark', fwmk])
```

Comment:
Please also set `fwmark` multiple times as its a multi node to test it's properly added into the backend

Reply:
Of course, "fwmark" should be a single node.
Because in the "ip rules" it will not check other options/sequence if the first "match" was matched.
I'll fix it.

Reply:
LeafNode "fwmark" was replaced as a single node.